src/plugins/vis_augmenter/public/vega/helpers.ts

```diff
@@ -350,14 +350,22 @@ export const addPointInTimeEventsLayersToSpec = (
  * Interaction handling functions mapped by their action names.
  */
 export const interactionHandlersByAction = {
-  [VisInteractionHandler.HANDLE_POINT_IN_TIME_CLICK]: (
+  [VisInteractionEventHandlerName.HANDLE_POINT_IN_TIME_CLICK]: (
     _event: ScenegraphEvent,
     item?: Item | null
   ) => {
     if (isPointInTimeAnnotation(item)) {
       // TODO: Show the events flyout
     }
   },
+  [VisInteractionEventHandlerName.HANDLE_POINT_IN_TIME_HOVER_IN]: (
```

Comment:
nit: same point here, if we can rename `HANDLE_POINT_IN_TIME_HOVER_IN` to include `EVENT` within the name, that can help clarify it.